Example/GiniVision/UIViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  UIViewController.swift
+//  GiniVision_Example
+//
+//  Created by Enrique del Pozo Gómez on 5/8/18.
+//  Copyright © 2018 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+import GiniVision
+
+extension UIViewController {
+    func showErrorDialog(for error: Error, positiveAction: (() -> Void)?) {
+        let message: String
+        var cancelActionTitle: String = NSLocalizedString("ginivision.camera.errorPopup.cancelButton",
+                                                          bundle: Bundle(for: GiniVision.self),
+                                                          comment: "cancel button title")
+        var confirmActionTitle: String? = NSLocalizedString("ginivision.camera.errorPopup.pickanotherfileButton",
+                                                            bundle: Bundle(for: GiniVision.self),
+                                                            comment: "pick another file button title")
+        
+        switch error {
+        case let validationError as DocumentValidationError:
+            message = validationError.message
+        case let customValidationError as CustomDocumentValidationError:
+            message = customValidationError.message
+        case let pickerError as FilePickerError:
+            message = pickerError.message
+            switch pickerError {
+            case .maxFilesPickedCountExceeded:
+                confirmActionTitle = NSLocalizedString("ginivision.camera.errorPopup.reviewPages",
+                                                       bundle: Bundle(for: GiniVision.self),
+                                                       comment: "review pages button title")
+            case .photoLibraryAccessDenied:
+                cancelActionTitle = NSLocalizedString("ginivision.camera.filepicker.errorPopup.cancelButton",
+                                                      bundle: Bundle(for: GiniVision.self),
+                                                      comment: "cancel button title")
+                confirmActionTitle = NSLocalizedString("ginivision.camera.filepicker.errorPopup.grantAccessButton",
+                                                       bundle: Bundle(for: GiniVision.self),
+                                                       comment: "cancel button title")
+            case .mixedDocumentsUnsupported:
+                cancelActionTitle = NSLocalizedString("ginivision.camera.mixedarrayspopup.cancel",
+                                                      bundle: Bundle(for: GiniVision.self),
+                                                      comment: "cancel button text for popup")
+                confirmActionTitle = NSLocalizedString("ginivision.camera.mixedarrayspopup.usePhotos",
```

Comment:
I'd prefer a tuple or dictionary of error to titles pairs. But just a suggestion.